src/features/commands.ts

```diff
@@ -19,11 +19,13 @@ const isWin = /^win/.test(process.platform);
 export default function registerCommands(server: OmnisharpServer, extensionPath: string) {
 	let d1 = vscode.commands.registerCommand('o.restart', () => server.restart());
 	let d2 = vscode.commands.registerCommand('o.pickProjectAndStart', () => pickProjectAndStart(server));
-	let d3 = vscode.commands.registerCommand('dotnet.restore', () => dotnetRestoreForAll(server));
-	let d4 = vscode.commands.registerCommand('o.showOutput', () => server.getChannel().show(vscode.ViewColumn.Three));
-    let d5 = vscode.commands.registerCommand('csharp.addTasksJson', () => addTasksJson(server, extensionPath));
+	let d3 = vscode.commands.registerCommand('o.restore', () => dnxRestoreForAll(server));
+	let d4 = vscode.commands.registerCommand('o.execute', () => dnxExecuteCommand(server));
+	let d5 = vscode.commands.registerCommand('o.execute-last-command', () => dnxExecuteLastCommand(server));
+	let d6 = vscode.commands.registerCommand('o.showOutput', () => server.getChannel().show(vscode.ViewColumn.Three));
+    let d7 = vscode.commands.registerCommand('csharp.addTasksJson', () => addTasksJson(server, extensionPath));
     
-	return vscode.Disposable.from(d1, d2, d3, d4, d5);
+	return vscode.Disposable.from(d1, d2, d3, d4, d5, d6);
```

Comment:
Looks like we are missing ',d7' here. Looks like the merge tool tried to be too smart.